spec/rubocop/cop/thread_safety/new_thread_spec.rb

```diff
@@ -45,6 +45,13 @@
     RUBY
   end
 
+  it 'registers an offense for starting a new thread with safe navigation' do
+    expect_offense(<<~RUBY)
+      Thread&.new { do_work }
+      ^^^^^^^^^^^ #{msg}
```

Comment:
That's weird

Reply:
Yeah, this construct does not make much sense in practice. But I thought that since technically this is still an issue to detect (`Thread.new` is the same as `Thread&.new` in runtime) it's easier to just do it